### DISCUSSION TITLE
Will be there a teleport docker image for v15.0.x? (no distroless)

### DISCUSSION THREAD

**Discussion by manamanna** [gravitational/teleport#38218] [Q&A] [answered]
Hi there,

at the moment I am missing a docker image for release 15.0.x. I saw there are ones for teleport-distroless but I need an image with a shell. Can anyone say when these will be published?

Thanks a lot!

**Comment by stevenGravy** [comment, marked as answer]
You can use the distroless-debug image if required. Please see https://goteleport.com/docs/installation/#docker. 15 and on no `teleport` or `teleport-ent` images will continue to be made as in the changelog. https://goteleport.com/docs/changelog/#container-images
